math/armstrong_number.cpp

```diff
@@ -28,16 +35,17 @@ int number_of_digits(int num) {
 }
 
 /**
- * Function to check whether the number is armstrong number or not.
- * @param num Number
- * @return `true` if the number is armstrong.
- * @return `false` if the number is not armstrong.
- */
+* @brief Function to check whether the number is armstrong number or not.
+* @param number to be checked
+* @return `true` if the number is armstrong.
+* @return `false` if the number is not armstrong.
+*/
```

Comment:
There should be spaces here (one of the last missing details).
```suggestion
 * @brief Function to check whether the number is armstrong number or not.
 * @param number to be checked
 * @return `true` if the number is armstrong.
 * @return `false` if the number is not armstrong.
 */
```